src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/ExtractToCodeBehindCodeActionResolverTest.cs

```diff
@@ -178,18 +186,19 @@ public async Task Handle_ExtractCodeBlockWithUsing()
             // Arrange
             var documentPath = "c:/Test.razor";
             var documentUri = new Uri(documentPath);
-            var contents = "@page \"/test\"\n@using System.Diagnostics\n@code { private var x = 1; }";
+            var contents = $"@page \"/test\"\n@using System.Diagnostics{Environment.NewLine}@code {{ private var x = 1; }}";
```

Comment:
Is the `{{ private var x = 1; }}` double curly brace intended?

Reply:
Yup 😄. In a template string (`$""`) if you want to write a curly brace you have to write two in order to opt-out of transitioning into C#

Reply:
Ahh makes sense, thanks!